client/config/config.go

```diff
@@ -65,112 +60,37 @@ func (c *ClientConfig) SetTrace(trace string) error {
 	c.Trace = boolVal
 	return nil
 }
+*/
 
-func DefaultClientConfig() *ClientConfig {
-	return &ClientConfig{chainID, keyringBackend, output, node, broadcastMode, trace}
-}
-
-// Cmd returns a CLI command to interactively create an application CLI
-// config file.
-func Cmd(defaultCLIHome string) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "config <key> [value]",
-		Short: "Create or query an application CLI configuration file",
-		RunE:  runConfigCmd,
-		Args:  cobra.RangeArgs(0, 2),
-	}
-
-	cmd.Flags().String(flags.FlagHome, defaultCLIHome,
-		"set client's home directory for configuration")
-	return cmd
-}
-
-func runConfigCmd(cmd *cobra.Command, args []string) error {
+func UpdateClientContextFromClientConfig(ctx client.Context) client.Context {
```

Comment:
UpdateClientContextFromClientConfig reads values from client.toml file and updates them in client Context.

Reply:
ReadFromClientConfig makes sense.